hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAdmin1.java

```diff
@@ -116,6 +130,8 @@ public void testCompactionTimestamps() throws Exception {
     assertEquals(0, ts);
 
     ADMIN.flush(tableName);
+    //ADMIN.majorCompactRegion(Bytes.toBytes("abcdefghijklmnopqrstuvwxyzabcdedgeghijklmnopqrs"));
+    ADMIN.majorCompactRegion(Bytes.toBytes("abcd"));
```

Comment:
Will remove these two lines, they are used for testing before the new test case.